cmd/client.go

```diff
@@ -0,0 +1,43 @@
+package cmd
+
+import (
+	"time"
+
+	"github.com/buoyantio/strest-grpc/client"
+	"github.com/spf13/cobra"
+)
+
+var clientCfg = client.Config{}
+
+var clientCmd = &cobra.Command{
+	Use:   "client",
+	Short: "run the strest-grpc client",
+	Args:  cobra.ArbitraryArgs,
```

Comment:
noticed the other commands use `cobra.NoArgs`? is this related to how we're doing `clientCfg.Run()` but also `server.Run()`. curious about the motivation for different patterns between client and server commands.

Reply:
Ah, this is a mistake on my part - it's supposed to be `NoArgs`